src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -566,7 +538,7 @@ $ npm install @sentry/apm
 
 #### Sending Traces
 
-To send traces, set the `tracesSampleRate` to a nonzero value. The following configuration will capture 25% of all your transactions:
+To send transaction, set the `tracesSampleRate` to a nonzero value. The following configuration will capture 25% of all your transactions:
```

Comment:
This section is titled "Sending Traces". I understood we mean "traces" in the more broad sense, for the page itself is about "Distributed Tracing".

I'd keep it "traces", because this configuration eventually determines not just that transactions from JS are sent, but that a group of transactions from all involved systems are sent -- thus forming a `trace` -- all transactions with the same `trace_id`.

```suggestion
To send traces, set the `tracesSampleRate` to a nonzero value. The following configuration will capture 25% of all your transactions:
```